app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -199,6 +200,10 @@ def curation_concern_from_search_results
 
       # Only returns unsuppressed documents the user has read access to
       def search_result_document(search_params)
+        # Don't filter any models when fetching a single result.
+        search_builder.class.default_processor_chain.delete :filter_models
```

Comment:
Will this remove `:filter_models` from the `Hyrax::WorkSearchBuilder` permanently at runtime? Is this okay for other users of the class? If so, could we get away with just dropping it from the initial processor chain to avoid changing the builder's behavior depending on whether this method has been called yet?